bluesky/utils/__init__.py

```diff
@@ -1551,7 +1551,16 @@ def block(self, blocking_event):
             # if with a Qt backend, do the scary thing
             if 'qt' in backend:
 
-                from matplotlib.backends.qt_compat import QtCore, QtWidgets
+                from matplotlib.backends.qt_compat import QtCore, QtWidgets, QT_API
+                import functools
+
+                @functools.lru_cache(None)
+                def _enum(name):
+                    # foo.bar.Enum.Entry (PyQt6) <=> foo.bar.Entry (non-PyQt6).
+                    return operator.attrgetter(
```

Comment:
This is a really fancy but hard to understand function, even though I can figure it out.
Worthy of at least a docstring or a potential refactor.

Reply:
I added a docstring.

Reply:
I'll say you did! 👏